_rules/inline_link_in_paragraph_is_distinguishable_be4d0c.md

```diff
@@ -0,0 +1,240 @@
+---
+id: be4d0c
+name: Inline link in paragraph is distinguishable
+rule_type: atomic
+
+description: |
+  This rule checks that links that are embedded in a paragraph have a way to distinguish them as links compared to the rest of the text
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:1.4.1: # Use of Color (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+
+acknowledgements:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+---
+
+## Applicability
+
+This rule applies to any HTML element which:
+
+- has the [semantic role][] of ['link'][link] or a [semantic role][] that inherits from the ['link'][link] role; and
+- is a [descendant][] of a `p` element; and
+- has [descendant][] [text nodes][text node] that are [visible][]; and
+- the `p` element has other [descendant][] [visible][] [text nodes][text node].
+
+## Expectation
+
+Each target element has either:
+
+- An icon immediately before, after or in the link denoting it as a link
+- Text immediately before, after or in the link denoting it as a link
+- a [distinguishing style][] compared to the other text not based on color alone
```

Comment:
The distinguishing style definition compares two elements, here you are comparing the link element to text nodes. I don't think that works. Do you mean to compare the link element to the parent element of the text node? If so, there are a bunch of problems with the distinguishing style definition that needs to be worked through. For example:

```html
<style>
  * { box-shadow: #F00 0 0 5px; color: black; }
  a { text-decoration: none; }
</style>
<p>Lorem ipsum <a href="">dolar sit amet</a> consectetur.</p>
```

This link is still distinguishable, because it's shadow is an inline one, whereas the shadow of the rest of the text is on the block.

Reply:
Updated the expectation (and assumptions) for 

- comparing elements instead of element and text nodes
- usage of border and box-shadow as distinguishing factors